packages/cdk/Makefile

```diff
@@ -12,7 +12,7 @@ format:
 	npm run format
 
 init: 
-	npm ci --silent
```

Comment:
This is to show failure message when using different versions of node

Reply:
I believe `--silent` will still show the final error message. Removing this makes the install a lot noisier

Reply:
Hmm I think I'm wrong here actually, though I still don't feel it's necessary to unsilence

Reply:
Done